src/content/docs/browser/new-relic-browser/browser-pro-features/browser-data-distributed-tracing.mdx

```diff
@@ -10,7 +10,9 @@ redirects:
   - /docs/distributed-tracing/ui-data/browser-data/
 ---
 
-import distributedTracingBrowserSpanIcon from 'images/distributed-tracing-browser-span-icon.png'
+import browserBrowserIcon from 'images/browser_icon_browser-icon.png'
+
+import browserErrorProfile from 'images/browser_screenshot-crop_error-profile.png'
```

Comment:
@a-sassman it looks like this import statement doesn't have a matching image tag in the doc itself